src/gui/userstatusselectormodel.cpp

```diff
@@ -406,23 +406,23 @@ QString UserStatusSelectorModel::timeDifferenceToString(int differenceSecs) cons
     } else if (differenceSecs < 60 * 60) {
         const auto minutesLeft = std::ceil(differenceSecs / 60.0);
         if (minutesLeft == 1) {
-            return tr("1 minute");
+            return tr("%1 minute", "", minutesLeft);
         } else {
-            return tr("%1 minutes").arg(minutesLeft);
+            return tr("%1 minutes", "", minutesLeft);
         }
     } else if (differenceSecs < 60 * 60 * 24) {
         const auto hoursLeft = std::ceil(differenceSecs / 60.0 / 60.0);
         if (hoursLeft == 1) {
-            return tr("1 hour");
+            return tr("%1 hour, "", hoursLeft);
         } else {
-            return tr("%1 hours").arg(hoursLeft);
+            return tr("%1 hours, "", hoursLeft);
         }
     } else {
         const auto daysLeft = std::ceil(differenceSecs / 60.0 / 60.0 / 24.0);
         if (daysLeft == 1) {
-            return tr("1 day");
+            return tr("%1 day", "", daysLeft);
         } else {
-            return tr("%1 days").arg(daysLeft);
+            return tr("%1 days", "", daysLeft);
```

Comment:
This change is different to line 418. See additional " char.

Reply:
@mgallien Is this incorrect encoding?

Reply:
@Valdnet I will check

Reply:
@mgallien Another request 😉.

Reply:
do not remove the `arg` call

Reply:
> This change is different to line 418. See additional " char.

@rakekniven Thanks for pointing out the error 😉. Only now did I notice that he was on line 418.